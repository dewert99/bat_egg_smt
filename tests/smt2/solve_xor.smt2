(declare-const a Bool)
(declare-const b Bool)
(declare-const c Bool)
(assert (xor a b c))
(check-sat-assuming (a (not (and a b c))))
(get-model)
(assert (xor (not a) (not b) (not c)))
(check-sat)