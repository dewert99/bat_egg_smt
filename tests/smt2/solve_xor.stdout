Sat
(
 (define-fun a () Bool true)
 (define-fun b () Bool false)
 (define-fun c () Bool false)
)
Unsat
